releasenotes/notes/features-5291b4198f98952c.yaml

```
@@ -0,0 +1,6 @@
+---
+
+features:
+  - |
+    Updating process agent discovery configuration to include a Data Scrubber for obfuscate sensitive information such as "passwords", keys, etc.
```

```suggestion
    Updating process agent discovery configuration to include a Data Scrubber for obfuscating sensitive information such as "passwords", keys, etc.
```
Leaving passwords in quotes since that seems to be intentional, normally I would omit the quotes.